pkg/test/componentlist.go

```diff
@@ -0,0 +1,55 @@
+package test
+
+import (
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+	"io"
+	"net/http"
+	"testing"
+)
+
+const (
+	kymaComponentListURL = "https://raw.githubusercontent.com/kyma-project/kyma/main/installation/resources/components.yaml"
```

Comment:
Why not get the file from `filepath.Join(workspace.InstallationResourceDir, "components.yaml")`?

Reply:
Would be an option, but this would require to clone the whole repository at the beginning of the unittest to finally use just one file of it. From resource perspective, it's cheaper to just pick the required file.

Reply:
But right - as the unit-tests which are using this feature are anyway cloning the repository up-front, it doens't make sense to download it here again.

Will change it to accept a path to the file as input-parameter.